xml/libvirt_configuration_virsh.xml

```diff
@@ -569,7 +569,32 @@ CPU Affinity:   yy
     See <link xlink:href="https://libvirt.org/pci-hotplug.html"/> for more
     information about PCI topology between &qemu; machine types.
     </para>
-  </important>
+   </important>
+
+   <sect2 xml:id="tip-libvirt-config-zpci">
+    <title>&pciback; for &zseries;</title>
+    <para>
+     &qemu; can configure zPCI attributes that are used when configuring &pciback;
+     for the &zseries; architecture. <option>uid</option> represents user-defined
+     identifier, while <option>fid</option> represents PCI function identifier.
+     These attributes are optional and if you do not specify them, they are
+     automatically generated with non-conflicting values.
+    </para>
+    <para>
+     These attributes are implemented by using a new PCI address extension flag.
+     To include zPCI attribute in your domain specification, use the following
+     example definition:
+    </para>
+<screen>
+&lt;hostdev mode='subsystem' type='pci'>
+ &lt;driver name='vfio'/>
+ &lt;source>
+  &lt;address domain='0x0' bus='0x03' slot='0x07' function='0x0'/&gt;
+ &lt;/source>
+ &lt;address type='pci' domain='0x0000' bus='0x00' slot='0x01' function='0x0' uid='0x0003' fid='0x00000027'/>
```

Comment:
I'm not familiar with s390, but this doesn't look right according to the libvirt domXML doc

https://libvirt.org/formatdomain.html#device-addresses

That text says "PCI address extensions depending on the architecture are supported. For example, PCI addresses for S390 guests will have a zpci child element, with two attributes: uid (a hex value between 0x0001 and 0xffff, inclusive), and fid (a hex value between 0x00000000 and 0xffffffff, inclusive) used by PCI devices on S390 for User-defined Identifiers and Function Identifiers."

As I understand it, zpci is a child element of  the address element. E.g.

<address type='pci' domain='0x0000' bus='0x00' slot='0x01' function='0x0'>
  <zpci uid='0x0003' fid='0x00000027'/>
</address>

And if I'm reading the schema correctly, it also defines a zpci element used by the address element

https://gitlab.com/libvirt/libvirt/-/blob/master/docs/schemas/basictypes.rng#L125
https://gitlab.com/libvirt/libvirt/-/blob/master/docs/schemas/domaincommon.rng#L6447

IMO we should also have @liayan review this change.